### DISCUSSION TITLE
Microphone permission not requested in Android browsers for OpenWebUI

### DISCUSSION THREAD

**Discussion by Sam8688** [open-webui/open-webui#3765] [General]
Description:
When attempting to use the microphone feature in OpenWebUI on Android, the website does not request microphone permissions from the browser, even though the browser itself has microphone permissions enabled. This issue occurs across multiple Android browsers including Brave, Duckduckgo, Croemite, and others.

Steps to Reproduce:
1. Open Brave browser (or any other Android browser).
2. Ensure that the browser has microphone permissions enabled in the app settings.
3. Navigate to the OpenWebUI IP address.
4. Click the microphone button.

Expected Behavior:
The OpenWebUI website should prompt the user to grant microphone permissions.
Actual Behavior:
No permission request is prompted, and the microphone feature cannot be accessed. An error message "Permission denied when accessing media devices" is displayed.

Screenshots:
![ed02d454-a275-4df3-9934-b84c91522030.jpeg](https://github.com/open-webui/open-webui/assets/93866029/04460118-392e-4f8b-8e84-1ffc6244b879)

)

Additional Information:
Operating System [ GrapheneOS ]
- Version: [14]
- Brave browser version: [1.67.123]
-Croemite version [126.0.6478.127]
- Duckduckgo version [5.206.1]
- OpenWebUI version: [v0.4.9]

Thanks for creating such a fantastic piece of software ❤️

**Comment by jmtatsch** [comment]
is url https encrypted? many browsers/os require that for giving access to the microphone.

**Comment by thiswillbeyourgithub** [comment]
Actually it is possible to add an exception to any URL but it's well hidden. I made a discussion post there but pinging didn't work so [linking here instead](https://github.com/open-webui/open-webui/discussions/4352)

This way I can use voice recognition / STT on all my devices including android
